tests/steps/other_v2_steps.py

```diff
@@ -1469,11 +1473,42 @@ def simulate_atc_failure(context, group, path, message):
             ]
         ]
     )
-    assert resp.would_succeed is False
     assert fail_path == path
     assert message in resp.failure_message
 
 
+@when("I make a new simulate request.")
+def make_simulate_request(context):
+    context.simulate_request = SimulateRequest(txn_groups=[])
+
+
+@then("I lift log limits on that simulate request.")
+def lift_log_limits_in_request(context):
+    context.simulate_request.lift_log_limits = True
```

Comment:
If not too much trouble I'd probably suggest changing to `allow_more_logging` here too...

Reply:
right, that should align with naming